src/main/flight/position.c

```diff
@@ -137,16 +140,40 @@ void calculateEstimatedAltitude(timeUs_t currentTimeUs)
     }
 #endif
 
-    if (ARMING_FLAG(ARMED) && !altitudeOffsetSet) {
+    if (ARMING_FLAG(ARMED) && !altitudeOffsetSetBaro) {
         baroAltOffset = baroAlt;
-        gpsAltOffset = gpsAlt;
-        altitudeOffsetSet = true;
-    } else if (!ARMING_FLAG(ARMED) && altitudeOffsetSet) {
-        altitudeOffsetSet = false;
+        altitudeOffsetSetBaro = true;
+    } else if (!ARMING_FLAG(ARMED) && altitudeOffsetSetBaro) {
+        altitudeOffsetSetBaro = false;
     }
+
     baroAlt -= baroAltOffset;
+
+    int goodGpsSats = 999;
```

Comment:
what's this?

Reply:
hey bakwc did you get my message about accel stick doesnt work in ALTHOLD mode  can you help?

Reply:
I haven't done it yet.. How do you want them to work? Just fly up / down when you move them up / down?

Reply:
yes sir thats what i would like ,and plus ive got some cool videos  i would like to send you been working on some things